js/client/tests/http/api-gharial-spec.js

```diff
@@ -608,4 +607,116 @@ describe('_api/gharial', () => {
 
   });
 
+  it('should check if the if-match header is working - positive', () => {
+    const examples = require('@arangodb/graph-examples/example-graph');
+    const exampleGraphName = 'knows_graph';
+    const vName = 'persons';
+    const eName = 'knows';
+    expect(db._collection(eName)).to.be.null; // edgec
+    expect(db._collection(vName)).to.be.null; // vertexc
+    const g = examples.loadGraph(exampleGraphName);
+    expect(g).to.not.be.null;
+    expect(db._collection(eName)).to.not.be.null;
+    expect(db._collection(vName)).to.not.be.null;
+
+    const key = 'bob';
+    const doc = db[vName].document(key);
+    const revision = doc._rev; // get a valid revision
+
+    let req = request.get(url + '/' + exampleGraphName + '/edge/' + vName + '/' + key , {
+      headers: {
+        'if-match': revision
+      }
+    });
+    expect(req.statusCode).to.equal(200);
+    expect(req.json.edge).to.deep.equal(doc);
+  });
+
+  it('should check if the if-match header is working - negative', () => {
+    const examples = require('@arangodb/graph-examples/example-graph');
+    const exampleGraphName = 'knows_graph';
+    const vName = 'persons';
+    const eName = 'knows';
+    expect(db._collection(eName)).to.be.null; // edgec
```

Comment:
does it have to be `to.be.null()` (i.e. with parentheses)??

Reply:
expect(null).to.be.null; // Recommended

found in http://www.chaijs.com/api/bdd/

Reply:
Ok!